test/watchCases/simple/multi-compiler/webpack.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = [{
+    name: "changing",
+    entry: "./index.js",
+    output: {
+        filename: "./bundle.js"
+    }
+}, {
+    name: "static",
+    entry: "./static-file.js",
+    output: {
+        filename: "./static.js"
+    }
+}];
```

Comment:
Could you add another config. And a watch step in which two configuration change at once.